src/App.js

```diff
@@ -1,13 +1,52 @@
 import React from 'react';
 import './App.css';
+import Button from "./components/Button";
+import Product from "./components/Product";
+import Tile from "./components/Tile";
+import bag1 from "./assets/bag_1.png";
+import bag2 from "./assets/bag_2.png";
+import bag3 from "./assets/bag_3.png";
+import bag4 from "./assets/bag_4.png";
+import brand from "./assets/brand.png";
+import our_story from "./assets/our_story.png";
+
+import {logDOM} from "@testing-library/react";
+
+
+
 
 function App() {
+
   return (
-      <h1>Handbags & Purses</h1>
-  );
-}
+      <>
+         <h1>Handbags & Purses</h1>
+         <nav>
+             <Button typeOfButton="button" name="to the collection" klikEvent={() => console.log("To the collection")}/>
+             <Button typeOfButton="button" name="shop all bags" klikEvent={() => console.log("Shop all bags")} />
+             <Button typeOfButton="button" name="pre-orders" klikEvent={() => console.log("Pre-orders")} onDisabled={true} />
```

Comment:
Misschien een goed idee om de anonieme functie mee te geven in de Button component zelf! Nu moet je 3x dezelfde code schrijven. Zelfde geld voor name. Deze schrijf je later weer uit in de console.log()